### PR TITLE
Vanilla Weapons Expanded

### PR DESCRIPTION
**Pull request by onigamii**
## Additions

Mostly updates the current broken VWE patch.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

**Comment by onigamii** [comment]
Pushed some changes, mostly made by Juan #6 and renamed the phosphate to fire foam for lore friendliness

File: Patches/Vanilla Weapons Expanded/Ammo/FireExtinguisher.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
           <value>
             <ThingCategoryDef>
               <defName>AmmoVWEExtinguisher</defName>
-              <label>flame suppressant</label>
+              <label>fire foam</label>
```
Comment (MadaraUchiha) [inline Patches/Vanilla Weapons Expanded/Ammo/FireExtinguisher.xml:19]: Actual term is "firefoam". See https://rimworldwiki.com/wiki/Firefoam_popper

File: Patches/Vanilla Weapons Expanded/Medieval_Ranged.xml [hunk 2]
```diff
@@ -201,7 +201,7 @@
 
         <!-- === Musket === -->
         <!-- 
-          - VWE uses a custom verb with to trigger smoke effects after the gun fires, this causes trouble with CE and just overriding it doesn't work.
+          - VWE uses a custom verb to trigger smoke effects after the gun is fired, this causes trouble with CE and just overriding it doesn"t work.
```
Comment (MadaraUchiha) [inline Patches/Vanilla Weapons Expanded/Medieval_Ranged.xml:204]: Your "replace all" doesn"t work well.

**Comment by N7Huntsman** [comment]
A few remarks from testing:
- The Charge Rocket Launcher has a fire mode selector, but probably shouldn't, so it can be consistent with the other rocket launchers.
- Projectile speed on the fire extinguisher visually appears to be pretty high.
- The hand cannon is the Chippa Rhino, and so should probably be swapped from .44 mag to .357 mag.
- Whip range is pretty high, higher than the range of a number of pistols. I might set it to 5 - 7.

**Comment by nomoretime7** [comment]
I have tested the patch and didn't encounter any errors.
have played for 1year (in game time) with about 110 mods
tested almost all of the weapons. and especially the bows and the sawed off shotgun
all seems ok for me

**Pull request merged**